frontend/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Apr 13 16:20:04 PDT 2017
+#Wed Aug 22 23:52:14 PDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.9-all.zip
```

Comment:
Please revert this file. This file will be changed automatically if you config your IntelliJ "use gradle wraper task", chose "use default gradle wrapper" option

Reply:
Done.